src/client/automation/settings.js

```diff
@@ -1,7 +1,4 @@
-export const MOVING_SPEED = 1000; //pixels/ms
-export const MOVING_SPEED_IN_DRAGGING = 4; //pixels/ms
-export const MINIMUM_MOVING_TIME = 25;
-
-//NOTE: some scripts work incorrectly when a delay between user actions too small, so we should make it more
-export const ACTION_STEP_DELAY      = 60;
-export const DRAG_ACTION_STEP_DELAY = 100;
+export default {
+    ACTION_STEP_DELAY:      10,
+    DRAG_ACTION_STEP_DELAY: 10
+};
```

Comment:
Export as separate constants as it was before. We export const objects only if it's enum.